src/components/TeamMember.js

```diff
@@ -1,30 +1,27 @@
-import {
-  Box,
-  Card,
-  CardContent,
-  Typography,
-} from "@material-ui/core/";
+import { Box, Card, CardContent, Typography } from "@material-ui/core/";
 import useMyStyles from "../styles/MyStyles";
 import AccountCircleSharpIcon from "@material-ui/icons/AccountCircleSharp";
 
 const TeamMember = (props) => {
   const classes = useMyStyles();
+  const { id, firstName, lastName, url, description } = props.userData;
 
   return (
-    <Card
-      key={props.id}
-      className={classes.card__TeamMember}
-      variant="outlined"
-    >
-      <CardContent className={classes.cardContent__TeamMember}>
-        <Box className={classes.cardBox__TeamMember}>
-          <Typography gutterBottom variant="h5" component="h2">
-            <AccountCircleSharpIcon color="primary" />{" "}
-            {`${props.firstName} ${props.lastName}`}
-          </Typography>
-          <img className={classes.teamMemberImage} src={props.url} alt="Specialist" />
+    <Card key={id} variant="outlined" style={{ marginBottom: ".2rem" }}>
+      <CardContent className={classes.TeamMember__mainWrapper}>
+        <Box className={classes.TeamMember__contentWrapper}>
+          <h2>
+            <p className={classes.TeamMember__name}>
+              <AccountCircleSharpIcon color="primary" />
+              {""}
```

Comment:
not needed I guess

Reply:
Removed <p> and moved className to h2 element.